components/image/image.html

```diff
@@ -1,66 +1,146 @@
-                <ul class="docs-sidebar_list text-14">
-                    <li class="docs-sidebar_list-item local-text-peach"><a href="../intro/intro.html">Introduction</a>
-                    </li>
-                    <li class="docs-sidebar_list-item local-text-peach"><a
-                            href="../typography/typography.html">Text Utilites</a></li>
-                </ul>
-            </div>
+        <div class="docs-sidebar_group">
+          <h2 class="text-general docs-sidebar_heading text-24">Components</h2>
+          <ul class="docs-sidebar_list text-16">
+            <li class="docs-sidebar_list-item">
+              <a href="../alert/alert.html">Alert</a>
+            </li>
+            <li class="docs-sidebar_list-item">
+              <a href="../avatar/avatar.html">Avatar</a>
+            </li>
+            <li class="docs-sidebar_list-item">
+              <a href="../badge/badge.html">Badge</a>
+            </li>
+            <li class="docs-sidebar_list-item">
+              <a href="../button/button.html">Button</a>
+            </li>
+            <li class="docs-sidebar_list-item">
+              <a href="../card/card.html">Card</a>
+            </li>
+            <li class="docs-sidebar_list-item-active">
+              <a href="./image.html">Image</a>
+            </li>
+            <li class="docs-sidebar_list-item">
+              <a href="../input/input.html">Input</a>
+            </li>
+            <li class="docs-sidebar_list-item">
+              <a href="../rating/rating.html">Rating</a>
+            </li>
+            <li class="docs-sidebar_list-item">
+              <a href="../input/input.html">Grid</a>
+            </li>
+          </ul>
+        </div>
+      </aside>
 
-            <div class="docs-sidebar_group">
-                <h2 class="text-general docs-sidebar_heading text-24">Components</h2>
-                <ul class="docs-sidebar_list text-16">
-                    <li class="docs-sidebar_list-item"><a href="../alert/alert.html">Alert</a></li>
-                    <li class="docs-sidebar_list-item"><a href="../avatar/avatar.html">Avatar</a></li>
-                    <li class="docs-sidebar_list-item"><a href="../badge/badge.html">Badge</a></li>
-                    <li class="docs-sidebar_list-item"><a href="../button/button.html">Button</a></li>
-                    <li class="docs-sidebar_list-item"><a href="../card/card.html">Card</a></li>
-                    <li class="docs-sidebar_list-item-active"><a href="./image.html">Image</a></li>
-                    <li class="docs-sidebar_list-item"><a href="../input/input.html">Input</a></li>
-                    <li class="docs-sidebar_list-item"><a href="../rating/rating.html">Rating</a></li>
-                    <li class="docs-sidebar_list-item"><a href="../input/input.html">Grid</a></li>
+      <div class="docs-content text-general">
+        <h1 class="text-36">Images</h1>
```

Comment:
can we use `text-size-36` or `font-size-lg` as a classname , it would be more readable

Reply:
Okay, I am just following that naming convention. Thanks for the suggestion. I'll refactor this soon